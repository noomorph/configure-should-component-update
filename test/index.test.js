const index = require('..');

describe('exported API', () => {
  it('should match the snapshot', () => {
    expect(index).toMatchSnapshot();
  });
});
