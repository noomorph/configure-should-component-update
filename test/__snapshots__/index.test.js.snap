// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exported API should match the snapshot 1`] = `
Object {
  "configureShouldComponentUpdate": [Function],
  "createStructuredEqualityComparer": [Function],
  "shallowEqual": Object {
    "arrays": [Function],
    "objects": [Function],
  },
  "strictEqual": [Function],
}
`;
